pkg/loop/config.go

```diff
@@ -11,13 +11,20 @@ import (
 )
 
 const (
-	envDatabaseURL            = "CL_DATABASE_URL"
-	envPromPort               = "CL_PROMETHEUS_PORT"
+	envDatabaseURL = "CL_DATABASE_URL"
+	envPromPort    = "CL_PROMETHEUS_PORT"
+
 	envTracingEnabled         = "CL_TRACING_ENABLED"
 	envTracingCollectorTarget = "CL_TRACING_COLLECTOR_TARGET"
 	envTracingSamplingRatio   = "CL_TRACING_SAMPLING_RATIO"
 	envTracingAttribute       = "CL_TRACING_ATTRIBUTE_"
 	envTracingTLSCertPath     = "CL_TRACING_TLS_CERT_PATH"
+
+	envBeholderEndpoint         = "CL_BEHOLDER_ENDPOINT"
+	envBeholderInsecureConn     = "CL_BEHOLDER_INSECURE_CONNECTION"
+	envBeholderCACertFile       = "CL_BEHOLDER_CA_CERT_FILE"
+	envBeholderAttribute        = "CL_BEHOLDER_ATTRIBUTE_"
+	envBeholderTraceSampleRatio = "CL_BEHOLDER_TRACE_SAMPLE_RATIO"
```

Comment:
Should other beholder config [options](https://github.com/smartcontractkit/chainlink-common/blob/main/pkg/beholder/config.go#L10-L30) be configurable ?